cmd/errtrace/main.go

```diff
@@ -194,6 +200,8 @@ type fileRequest struct {
 	Write    bool
 	List     bool
 	Filename string
+
+	ImplicitStdin bool
```

Comment:
we're writing to this variable, but never using it -- did you mean to check it before printing the "reading from stdin" print? or should we drop the var?

Reply:
Derp, I meant to use this.
I want to print the warning only for implicit stdin because that's the common case for "I'm just running the command to see how to use it."